server/tests-py/test_v1_queries.py

```diff
@@ -168,6 +168,18 @@ def test_city_where_similar(self, hge_ctx):
     def test_city_where_not_similar(self, hge_ctx):
         check_query_f(hge_ctx, self.dir() + '/select_city_where_not_similar.yaml')
 
+    def test_city_where_posix_cs(self, hge_ctx):
+        check_query_f(hge_ctx, self.dir() + '/select_city_where_posix_cs.yaml')
+
+    def test_city_where_not_posix_cs(self, hge_ctx):
+        check_query_f(hge_ctx, self.dir() + '/select_city_where_not_posix_cs.yaml')
+
+    def test_city_where_posix_ci(self, hge_ctx):
+        check_query_f(hge_ctx, self.dir() + '/select_city_where_posix_ci.yaml')
+
+    def test_city_where_not_posix_ci(self, hge_ctx):
+        check_query_f(hge_ctx, self.dir() + '/select_city_where_not_posix_ci.yaml')
+
```

Comment:
Few test cases to use one of the operators in select, update, insert and delete permission would be very useful.

Reply:
Something which checks for valid emails maybe?

Reply:
Actually one select permission test is enough.

Reply:
Could you link me an exemple of such a test with another operator?

Reply:
This one has a select permission check for `_in` operator: https://github.com/hasura/graphql-engine/blob/29925eb08de0cbe373f90f6ca6a21d19780a486a/server/tests-py/queries/graphql_query/permissions/setup.yaml#L248

You can perhaps add one in this file itself?

Reply:
Is that ok? https://github.com/hasura/graphql-engine/pull/6172/commits/d14a419298ce2c977024b9f9c4c5efdc459f52be